python/cuml/manifold/umap.pyx

```diff
@@ -289,6 +290,11 @@ class UMAP(UniversalBase,
         type. If None, the output type set at the module level
         (`cuml.global_settings.output_type`) will be used. See
         :ref:`output-data-type-configuration` for more info.
+    build_algo: string (default='brute_force_knn')
+        How to build the knn graph. Supported build algorithms are ['brute_force_knn',
+        'nn_descent']
+    metric_kwds: dict (optional, default=None)
+        Build algorithm argument
```

Comment:
I think you meant `build_kwds` here. Would be nice to detail the parameters and their default values.